X_train:  age                  0
workclass         1836
education_num        0
marital_status       0
occupation        1843
relationship         0
race                 0
sex                  0
capital_change       0
hours_per_week       0
native_country       0
dtype: int64
Index(['workclass', 'occupation'], dtype='object')
Traceback (most recent call last):
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular/train.py", line 61, in <module>
    loss = criterion(outputs, batch_y)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/torch/nn/modules/loss.py", line 603, in forward
    return F.binary_cross_entropy(input, target, weight=self.weight, reduction=self.reduction)
  File "/home/andrea/PycharmProjects/data_exfiltration_tabular_venv/lib/python3.6/site-packages/torch/nn/functional.py", line 2908, in binary_cross_entropy
    "Please ensure they have the same size.".format(target.size(), input.size())
ValueError: Using a target size (torch.Size([64])) that is different to the input size (torch.Size([64, 1])) is deprecated. Please ensure they have the same size.